www/lobby/index.ts

```diff
@@ -41,5 +41,8 @@ lobby_router.post('/', async (req: Request, res: Response) => {
  * Body Params: id, refreshToken
  */
 lobby_router.get('/', async (req: Request, res: Response) => {
-  res.status(200).json('placeholder get');
+  const userId = req.body.id;
```

Comment:
im guessing you tested this and the validator works?

Reply:
which validator?

Reply:
the `validateUserJwt`..